toontown/ai/QuestManagerAI.py

```diff
@@ -584,6 +584,8 @@ def quests(command, arg0=0, arg1=0):
         if arg0:
             if canCarry:
                 if arg0 in Quests.QuestDict.keys():
+                    quest = Quests.getQuest(arg0)
+                    invoker.addQuest(quest, 0)                    
```

Comment:
What was the intention of this change?

Reply:
To get quests add doing something instead of nothing
Basically adding quests to the next available slot in ur task slots

Reply:
This doesn't work.